### ISSUE TITLE
Github action doesn't work

### ISSUE THREAD

**Issue by topshik** [mhitza/flake8-jupyter-notebook#1]
Hi.

I've just made a simple repository https://github.com/topshik/jupyter-test-linter where I wanted to test your linter with a simple example: create a branch with a linter and .ipynb file, so that it can be checked by a linter. However, the step with applying your code always fails with some JS memory problems. 

```
<--- Last few GCs --->
FATAL ERROR: Ineffective mark-compacts near heap limit Allocation failed - JavaScript heap out of memory

[18:0x5582cf6f7f80]    65724 ms: Scavenge 2046.7 (2050.7) -> 2045.7 (2050.7) MB, 2.0 / 0.0 ms  (average mu = 0.235, current mu = 0.213) allocation failure 
[18:0x5582cf6f7f80]    65805 ms: Scavenge 2046.8 (2050.7) -> 2045.8 (2049.7) MB, 14.1 / 0.0 ms  (average mu = 0.235, current mu = 0.213) allocation failure 
[18:0x5582cf6f7f80]    65931 ms: Scavenge 2046.8 (2049.7) -> 2045.8 (2049.7) MB, 9.6 / 0.0 ms  (average mu = 0.235, current mu = 0.213) allocation failure 


<--- JS stacktrace --->
 1: 0x7f764c129d4c node::Abort() [/lib64/libnode.so.72]

==== JS stack trace =========================================

    0: ExitFrame [pc: 0x7f764cf884b9]
Security context: 0x371f82e1b161 <JSObject>
    1: match [0x371f82e09891](this=0x043d40eb6f29 <String[18]:    "metadata": {},>,0x159ee930f5c9 <JSRegExp <String[#18]: ^\s{3}"cell_type":>>)
    2: find_source_blocks [0x1032084a00f1] [/annotate:~140] [pc=0x3055ab189746](this=0x346d9066b999 <JSGlobal Object>,0x043d40eb64b1 <JSArray[67]>)
    3: /* anonymous */ [0x1032084a0131] [/annotate:27] [bytecode=0x1...

 2: 0x7f764bf638cd node::OnFatalError(char const*, char const*) [/lib64/libnode.so.72]
 3: 0x7f764c48cc7a v8::Utils::ReportOOMFailure(v8::internal::Isolate*, char const*, bool) [/lib64/libnode.so.72]
 4: 0x7f764c48cf02 v8::internal::V8::FatalProcessOutOfMemory(v8::internal::Isolate*, char const*, bool) [/lib64/libnode.so.72]
 5: 0x7f764c60cd89  [/lib64/libnode.so.72]
 6: 0x7f764c621717 v8::internal::Heap::PerformGarbageCollection(v8::internal::GarbageCollector, v8::GCCallbackFlags) [/lib64/libnode.so.72]
 7: 0x7f764c62247a v8::internal::Heap::CollectGarbage(v8::internal::AllocationSpace, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/lib64/libnode.so.72]
 8: 0x7f764c6227e0 v8::internal::Heap::CollectAllGarbage(int, v8::internal::GarbageCollectionReason, v8::GCCallbackFlags) [/lib64/libnode.so.72]
 9: 0x7f764c5d77b2 v8::internal::StackGuard::HandleInterrupts() [/lib64/libnode.so.72]
10: 0x7f764c8fb1f5 v8::internal::Runtime_StackGuard(int, unsigned long*, v8::internal::Isolate*) [/lib64/libnode.so.72]
11: 0x7f764cf884b9  [/lib64/libnode.so.72]
/entrypoint.sh: line 3:    18 Aborted                 (core dumped) /annotate
```

**Comment by mhitza** [comment]
The issue seems to be caused by the empty source block, which is unexpected in code and leads to an infinite loop. Seems like I'll need to handle that case in the future. My current question is if the notebook file has been exported, or somehow manually written/altered?

It's been a year since I wrote this action, for an internal project, and at that time I based it on existing examples I had on hand, and an empty source block (`"source": []`) wasn't something I've encountered.
